### PR TITLE
Minor changes for eltwise on Grace

### PR DESCRIPTION
**Pull request by stefan0re**
Now the eltwise functions also run with BF16 on Grace.

### CODE REVIEW

File: samples/eltwise/run_test_neov2.txt [hunk 1]
```diff
@@ -1,453 +1,453 @@
-binary_add_bf16_f32_bf16_f32_eqld
```
Comment (alheinecke) [inline samples/eltwise/run_test_neov2.txt:1]: Is this file differently sorted? I have a hard time following the changes hence asking. We sort the files alphanumerically.

**Review by alheinecke — COMMENTED** [review]
Minor nit

File: samples/eltwise/run_test_neov2.txt (hunk 1 above)
Comment (stefan0re) [reply samples/eltwise/run_test_neov2.txt:1]: Not really, these are the tests I have added:
unary_quant_f32_i16_eqld
unary_quant_f32_i16_gtld
unary_quant_f32_i32_eqld
unary_quant_f32_i32_gtld
unary_reduce_absmax_cols_x_overwrite_bf8_eqld
unary_reduce_absmax_cols_x_overwrite_bf8_gtld
unary_reduce_absmax_cols_x_overwrite_f16_eqld
unary_reduce_absmax_cols_x_overwrite_f16_gtld
unary_reduce_absmax_cols_x_overwrite_hf8_eqld
unary_reduce_absmax_cols_x_overwrite_hf8_gtld
unary_reduce_absmax_rows_x_overwrite_bf8_eqld
unary_reduce_absmax_rows_x_overwrite_bf8_gtld
unary_reduce_absmax_rows_x_overwrite_f16_eqld
unary_reduce_absmax_rows_x_overwrite_f16_gtld
unary_reduce_absmax_rows_x_overwrite_hf8_eqld
unary_reduce_absmax_rows_x_overwrite_hf8_gtld
unary_transform_norm_to_vnni8_bf16_eqld
unary_transform_norm_to_vnni8_bf16_gtld
unary_transform_norm_to_vnni8_bf8_eqld
unary_transform_norm_to_vnni8_bf8_gtld
unary_transform_norm_to_vnni8_f16_eqld
unary_transform_norm_to_vnni8_f16_gtld
unary_transform_norm_to_vnni8_hf8_eqld
unary_transform_norm_to_vnni8_hf8_gtld
unary_transform_norm_to_vnni8_i16_eqld
unary_transform_norm_to_vnni8_i16_gtld
unary_transform_norm_to_vnni8_i8_eqld
unary_transform_norm_to_vnni8_i8_gtld
unary_transform_norm_to_vnni8t_bf16_eqld
unary_transform_norm_to_vnni8t_bf16_gtld
unary_transform_norm_to_vnni8t_f16_eqld
unary_transform_norm_to_vnni8t_f16_gtld
unary_transform_norm_to_vnni8t_i16_eqld
unary_transform_norm_to_vnni8t_i16_gtld
unary_transform_vnni8_to_vnni8t_bf16_eqld
unary_transform_vnni8_to_vnni8t_bf16_gtld
unary_transform_vnni8_to_vnni8t_bf8_eqld
unary_transform_vnni8_to_vnni8t_bf8_gtld
unary_transform_vnni8_to_vnni8t_f16_eqld
unary_transform_vnni8_to_vnni8t_f16_gtld
unary_transform_vnni8_to_vnni8t_hf8_eqld
unary_transform_vnni8_to_vnni8t_hf8_gtld
unary_transform_vnni8_to_vnni8t_i16_eqld
unary_transform_vnni8_to_vnni8t_i16_gtld
unary_transform_vnni8_to_vnni8t_i8_eqld
unary_transform_vnni8_to_vnni8t_i8_gtld
unary_transform_vnni8t_to_norm_bf16_eqld
unary_transform_vnni8t_to_norm_bf16_gtld
unary_transform_vnni8t_to_norm_f16_eqld
unary_transform_vnni8t_to_norm_f16_gtld
unary_transform_vnni8t_to_norm_i16_eqld
unary_transform_vnni8t_to_norm_i16_gtld

**Comment by alheinecke** [comment]
We will merge #926 soon. With that my comment becomes obsolete as the file goes away... please merge main after #926 landed and then we are good to go here.

**Comment by alheinecke** [comment]
please merge again.

**Pull request closed without merging**